_stbt/core.py

```diff
@@ -1698,6 +1698,7 @@ def __init__(self, user_sink_pipeline, raise_in_user_thread, save_video=""):
         self.annotations = []
         self._raise_in_user_thread = raise_in_user_thread
         self.received_eos = threading.Event()
+        self._frames = deque(maxlen=35)
```

Comment:
Does this assume a particular video-capture frame rate?

Reply:
I suppose so.  This is just over 0.5s at 60fps.  I wanted to put a limit in to prevent runaway memory usage.  This will be 96MB of video frame data.